### PR TITLE
Apply navigation view design

### LINKED ISSUE #439: Apply navigation view design
## Kind (Required)

- Improvement

 ## Overview (Required)

- We can change padding between icon and title.
- We can change some icons.

 ## Screenshot	
Currently | Design
:--: | :--:
 |

### PR DESCRIPTION
**Pull request by nano-nano**
## Issue
- close #439 

## Overview (Required)
- replace some drawer's icon
- adjust padding between icon and title
- replace README.md's screen capture (because Features capture in README.md has drawer screen)

## Links
- None

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: corecomponent/androidcomponent/src/main/res/menu/menu_nav_drawer.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
         >
         <item
             android:id="@id/main"
-            android:icon="@drawable/ic_event_outline_black_24dp"
+            android:icon="@drawable/ic_event_24px"
```
Comment (nano-nano) [inline corecomponent/androidcomponent/src/main/res/menu/menu_nav_drawer.xml]: Drawable file's name referred to Figma's content.
https://www.figma.com/file/4r9becvhDy3GfXaXex8E8d/App?node-id=21%3A543

Should I rename other drawable file's name?
ex. `ic_android_black_24dp.xml` -> `ic_android_24dp.xml`

**Comment by nano-nano** [comment]
Drawable file's name referred to Figma's content.
https://www.figma.com/file/4r9becvhDy3GfXaXex8E8d/App?node-id=21%3A543

Should I rename other drawable file's name?
ex. ic_android_black_24dp.xml -> ic_android_24dp.xml

File: corecomponent/androidcomponent/src/main/res/drawable/ic_settings_24px.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
...,11.02C19.4722,11.34 19.5022,11.66 19.5022,12ZM17.5022,12C17.5022,11.79 17.4922,11.58 17.4522,11.27L17.3122,10.14L18.2022,9.44L19.2722,8.59L18.5722,7.38L17.3022,7.89L16.2422,8.32L15.3322,7.62C14.9322,7.32 14.5322,7.09 14.1022,6.91L13.0422,6.48L12.8822,5.35L12.6922,4H11.3022L11.1022,5.35L10.9422,6.48L9.8822,6.91C9.4722,7.08 9.0622,7.32 8.6322,7.64L7.7322,8.32L6.6922,7.9L5.4222,7.39L4.7222,8.6L5.8022,9.44L6.6922,10.14L6.5522,11.27C6.5222,11.57 6.5022,11.8 6.5022,12C6.5022,12.2 6.5222,12.43 6.5522,12.74L6.6922,13.87L5.8022,14.57L4.7222,15.41L5.4222,16.62L6.6922,16.11L7.7522,15.68L8.6622,16.38C9.0622,16.68 9.4622,16.91 9.8922,17.09L10.9522,17.52L11.1122,18.65L11.3022,20H12.7022L12.9022,18.65L13.0622,17.52L14.1222,17.09C14.5322,16.92 14.9422,16.68 15.3722,16.36L16.2722,15.68L17.3122,16.1L18.5822,16.61L19.2822,15.4L18.2022,14.56L17.3122,13.86L17.4522,12.73C17.4822,12.43 17.5022,12.21 17.5022,12ZM12.0022,8C9.7922,8 8.0022,9.79 8.0022,12C8.0022,14.21 9.7922,16 12.0022,16C14.2122,16 16.0022,14.21 16.0022,12C16.0022,9.79 14.2122,8 12.0022,8ZM10.0022,12C10.0022,13.1 10.9022,14 12.0022,14C13.1022,14 14.0022,13.1 14.0022,12C14.0022,10.9 13.1022,10 12.0022,10C10.9022,10 10.0022,10.9 10.0022,12Z"
```
[the remark this answers is not in the document]
Comment (nano-nano) [reply corecomponent/androidcomponent/src/main/res/drawable/ic_settings_24px.xml]: I tried to reduce vector path using [avocado](https://github.com/alexjlockwood/avocado) in my local environment.
But its warning still remained.😢
What can I do to fix it...?

[the remark this answers is not in the document]
Comment (jmatsu) [reply corecomponent/androidcomponent/src/main/res/drawable/ic_settings_24px.xml]: @nano-nano 
As far as I know, avocado's optimization doesn't mean minimizing the path length. Anyway, we can ignore this warning. Great work!

[the remark this answers is not in the document]
Comment (takahirom) [reply corecomponent/androidcomponent/src/main/res/drawable/ic_settings_24px.xml]: If you need to use a long path, you can use png.
At this time ok!

[the remark this answers is not in the document]
Comment (nano-nano) [reply corecomponent/androidcomponent/src/main/res/drawable/ic_settings_24px.xml]: @jmatsu @takahirom 
Thank you for comments!

> As far as I know, avocado's optimization doesn't mean minimizing the path length

I see. It was good experience to me. 👀

**Pull request merged**